File: TheSwitcher_ObjC/TheSwitcher_ObjCTests/FormattingTests.m
//
//  FormattingTests.m
//  TheSwitcher_ObjcTests
//
//  Created by Fábio Maciel de Sousa on 13/02/21.
//

#import <XCTest/XCTest.h>
#import "Formatting.h"

@interface FormattingTests : XCTestCase

@end

@implementation FormattingTests

- (void)testStringFormatting {
    NSString *sut = @"Kitchen";
    XCTAssertTrue([[sut formattedDivision] isEqual:@"Your Kitchen light is"]);
}

@end

File: TheSwitcher_FabioMs/TheSwitcher_FabioMsTests/FormattingTests.swift
//
//  FormattingTests.swift
//  TheSwitcher_FabioMsTests
//
//  Created by Fábio Maciel de Sousa on 12/02/21.
//

import XCTest
@testable import TheSwitcher_FabioMs

class FormattingTests: XCTestCase {

    func testStringFormatting() {
        let sut = "Kitchen"
        XCTAssertEqual(sut.formattedDivision, "Your Kitchen light is")
    }

}
